src/features/ModelPlan/ModelToOperations/index.tsx

```diff
@@ -49,11 +48,7 @@ const ModelToOperations = () => {
         ]}
       />
 
-      {message && (
-        <Alert type="error" slim>
-          {message}
-        </Alert>
-      )}
+      {message}
```

Comment:
Removed the error alert hard coded here, so that other messages can be rendered here

Reply:
Milestone success message:
![image](https://github.com/user-attachments/assets/79fd3bab-436d-4723-ad74-e6101dfc6b79)


MTO Table error message:
![image](https://github.com/user-attachments/assets/c01036a3-b7de-4310-98c7-2536644bb30d)